lib/gettext_i18n_rails.rb

```diff
@@ -2,6 +2,15 @@
 require 'gettext_i18n_rails/gettext_hooks'
 
 module GettextI18nRails
+  IGNORE_TABLES = [/^sitemap_/, /_versions$/, 'schema_migrations', 'sessions', 'delayed_jobs']
+
+  def self.ignore_tables
+    IGNORE_TABLES
+  end
+
+  def self.add_ignore_table(str)
```

Comment:
A constant would work too.

My thought process for this was to have the ability to add new tables to the list without caring/knowing about the defaults. This can be achieved by calling the self.add_ignore_table(str) method in an initializer.